test/14-config/bare-metal.json

```diff
@@ -0,0 +1,5 @@
+{
+  "db": {
+    "name": "bare-metal-config-env-provided"
+  }
+}
```

Comment:
Looks like you've removed the final newline in these files. Could you un-do those changes?

Reply:
Yes, i will update this